### PR TITLE
Fixed console tab not scrolling to end on page load #1335

### PR DESCRIPTION
**Pull request by maheshp**

### CODE REVIEW

**Comment by ketan** [comment]
@maheshp - to be clear this JS kicks off on the console page as well as the failures tab, so make sure that the selector works there as well.

**Comment by maheshp** [comment]
@ketan verified both on console and failures tab, it looks ok

File: server/webapp/WEB-INF/rails.new/app/assets/javascripts/build_output_observer.js [hunk 1]
```diff
@@ -31,12 +31,16 @@ BuildOutputObserver.prototype = {
...
         this.autoScrollButton = jQuery('.auto-scroll');
         this.autoScrollButton.toggleClass('tailing', this.enableTailing);
         this.autoScrollButton.on('click', function(){
             self.enableTailing = !self.enableTailing;
             self.initializeScroll();
         });
+        this.consoleTabElement.on('click', function(){
+            self.initializeScroll();
+        });
         this.initializeScroll();
```
Comment (zabil) [inline server/webapp/WEB-INF/rails.new/app/assets/javascripts/build_output_observer.js]: you can shorten this to 
```
this.consoleTabElement.on('click', self.initializeScroll);
```
